src/components/figures/figure/figure.ts

```diff
@@ -180,8 +181,14 @@ export class FigureComponent extends Component<Props, SpreadsheetChildEnv> {
   }
 
   onKeyDown(ev: KeyboardEvent) {
-    const figure = this.props.figure;
+    // Allow the event to bubble to the grid only if Ctrl + Y or Ctrl + Z is pressed
+    // Let the grid handle the undo/redo operations
+    if (isCtrlKey(ev) && (ev.key.toUpperCase() === "Y" || ev.key.toUpperCase() === "Z")) {
+      return;
+    }
```

Comment:
```suggestion
    if (["CTRL+Z", "CTRL+Y"].includes(keyboardEventToShortcutString(ev)) {
      return;
    }
```
Nitpick. Use whichever one you find more readable

Reply:
Definitely, this one looks more readable :)